### ISSUE TITLE
Editing a message with an embed and attachment makes the attachment appear twice

### ISSUE THREAD

**Issue by anna-is-cute** [serenity-rs/serenity#757]
1. Create a message and send it to a channel. The message should have an embed that uses an attachment for its image.
2. Get the message by its ID (`get_message`).
3. Take the `Embed` from the `Message` (`message.embeds.remove(0)`).
4. Make a `CreateEmbed` from the `Embed` (`CreateEmbed::from(embed)`).
5. Add a field to the embed.
6. Edit the message (`edit_message`).
    ```rust
    channel.edit_message(http, message_id, |msg| msg.embed(|e| { *e = embed; e } ))?;
    ```
7. Notice the output looks something like ![this][image].

There was relevant discussion [on Discord][disc] prior to making this issue (see link).

Actual code:
  - [Making the message][making]
  - [Editing the embed][editing]

Serenity version: 0.7.4 from crates.io

[image]: https://cdn.discordapp.com/attachments/381912587505500160/658717349788123180/unknown.png
[disc]: https://discordapp.com/channels/381880193251409931/381912587505500160/658717335456317490
[making]: https://git.sr.ht/~jkcclemens/lalaworld/tree/temp/src/route/giveaways.rs#L399
[editing]: https://git.sr.ht/~jkcclemens/lalaworld/tree/temp/src/task.rs#L160

**Comment by steelx** [comment]
I did this. this is what I got:
```rust
msg.channel_id.send_message(&ctx.http, |m| {
        m.embed(|e| e
            .colour(0x00ff00)
            .thumbnail(profile.avatar.clone())
            .title(format!("Stats for {}", userid))
            .field("💁 K/D", format!("{}/{}", player_stats.kills, player_stats.deaths), false)
            .field("🖥 MMR", player_stats.mmr, false)
            .field("Max MMR", player_stats.max_mmr, false)
            .field("Rank", player_stats.rank, false)
            .image("https://lalafell.world/assets/gift.png")
            .footer(|f| {
                f.text("🕒 Stats last updated @");
                f
            })
            .timestamp(&last_updated_time)
            .author(|f| {
                f.icon_url(profile.avatar)
            })
        )
    }).await?;
```

![image](https://user-images.githubusercontent.com/3184210/91630690-15c76280-e9f1-11ea-89c6-60a5ae3e789c.png)

**Comment by Lakelezz** [comment]
As this uses Serenity 0.7 I will close this issue.
It may be fixed in 0.8/0.9, we do not support any older version than 0.8.
